### PR TITLE
[schema] Add next hop group table to schema

### PR DESCRIPTION
**Pull request by TACappleman**

### CODE REVIEW

**Comment by anshuv-mfst** [comment]
Hi @adamyeung - could you please add BRCM team to review this PR, thanks.

File: common/schema.h [hunk 1]
```diff
@@ -39,6 +39,7 @@ namespace swss {
 #define APP_FDB_TABLE_NAME                "FDB_TABLE"
 #define APP_PFC_WD_TABLE_NAME             "PFC_WD_TABLE"
 #define APP_SWITCH_TABLE_NAME             "SWITCH_TABLE"
+#define APP_NEXT_HOP_GROUP_TABLE_NAME	  "NEXT_HOP_GROUP_TABLE"
```
Comment (prsunny) [inline common/schema.h:42]: please fix alignment

Comment (TACappleman) [reply common/schema.h]: Fixed

File: common/schema.h [hunk 2]
```diff
@@ -39,6 +39,7 @@ namespace swss {
 #define APP_FDB_TABLE_NAME                "FDB_TABLE"
 #define APP_PFC_WD_TABLE_NAME             "PFC_WD_TABLE"
 #define APP_SWITCH_TABLE_NAME             "SWITCH_TABLE"
+#define APP_NEXT_HOP_GROUP_TABLE_NAME     "NEXT_HOP_GROUP_TABLE"
```
[the remark this answers is not in the document]
Comment (TACappleman) [reply common/schema.h]: Changed

**Review by prsunny — APPROVED** [review]

**Comment by TACappleman** [comment]
@prsunny any idea what's causing the failing checks and if they'll work with a retry? I'm pretty confident that my 1 line addition of a #define isn't causing the issues!

**Review by qiluo-msft — APPROVED** [review]

**Comment by qiluo-msft** [comment]
Could you rebase your code to latest master?

**Comment by TACappleman** [comment]
@qiluo-msft Done

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**